src/admin-views/editor/fieldset/price.php

```diff
@@ -34,8 +34,17 @@ class="price tribe-dependent"
 			class="ticket_field ticket_form_right"
 			size="7"
 			value="<?php echo esc_attr( $ticket ? $price : null ); ?>"
+			<?php if ( $provider instanceof Tribe__Tickets__Commerce__PayPal__Main ) {
+				echo 'data-validation-is-greater-than="0"',
+					'data-validation-error="' . esc_attr__( 'PayPal tickets require a non 0 price', 'event-tickets' ) . '"';
```

Comment:
nitpick but we should use the word `nonzero` here instead of `non 0`

Reply:
I've filled in that wording just to have something and am waiting from strategy to update, but thanks!